Podfile.lock

```diff
@@ -172,6 +173,6 @@ SPEC CHECKSUMS:
   YapDatabase: b1e43555a34a5298e23a045be96817a5ef0da58f
   ZXingObjC: bf15b3814f7a105b6d99f47da2333c93a063650a
 
-PODFILE CHECKSUM: 48dbf2fb380d626bb799a782dd41b6bf1e466506
+PODFILE CHECKSUM: 367862c422b92ee0f62ed16e3f0480d1510c9f50
 
-COCOAPODS: 1.1.1
```

Comment:
Sorry, but can you update to cocoapods 1.1.1 (or newer, and I'll follow suit). I'm wondering if that's why our xcode proj was re-written in this commit (which I don't really like). Maybe I'm crazy, but I actually try to understand the diffs to the xcode.proj.